meta/main.yml

```
@@ -21,22 +21,23 @@ galaxy_info:
   min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    - name: Amazon Linux 2
-      versions:
-        - any
+  # Amazon Linux 2023 does not currently offer yum packages
```

Amazon Linux 2023 uses `dnf`, not `yum`:
```suggestion
  # Amazon Linux 2023 does not currently offer system packages
```

---

Resolved in favor of [this comment](https://github.com/cisagov/ansible-role-automated-security-updates/pull/30#discussion_r1189071878).